crates/ruma-common/src/events/room/power_levels.rs

```diff
@@ -284,6 +300,55 @@ impl From<RedactedRoomPowerLevelsEventContent> for RoomPowerLevels {
     }
 }
 
+impl From<RoomPowerLevels> for RoomPowerLevelsEventContent {
+    fn from(c: RoomPowerLevels) -> Self {
+        Self {
+            ban: c.ban,
+            events: c.events,
+            events_default: c.events_default,
+            invite: c.invite,
+            kick: c.kick,
+            redact: c.redact,
+            state_default: c.state_default,
+            users: c.users,
+            users_default: c.users_default,
+            notifications: c.notifications,
+        }
+    }
+}
```

Comment:
Can we have *only* this impl? `.to_content()` seems a little bit too magic to me, I'd like users to have to go through `.power_levels()` first. Also if we decide to have `.to_content()` in the future, the separate `PowerLevels` type doesn't really make sense anymore any should be removed.